unity-client/Assets/Scripts/MainScripts/DCL/Components/Video/Tests/VideoTests.cs

```diff
@@ -163,15 +165,63 @@ public IEnumerator VideoTextureVisibleStateIsSetCorrectlyWhenEntityIsRemoved()
             yield return ent1Shape.routine;
 
             TestHelpers.SharedComponentAttach(ent1Shape, ent1);
-            yield return null; //a frame to wait DCLVideoTexture update    
+            yield return null; //a frame to wait DCLVideoTexture update
             Assert.IsTrue(videoTexture.isVisible, "DCLVideoTexture should be visible");
 
             scene.RemoveEntity(ent1.entityId, true);
-            yield return null; //a frame to wait DCLVideoTexture update            
+            yield return null; //a frame to wait DCLVideoTexture update
 
             Assert.IsTrue(!videoTexture.isVisible, "DCLVideoTexture should not be visible ");
         }
 
+        [UnityTest]
+        public IEnumerator VolumeIsToggledWhenCharacterEntersOrLeavesScene()
```

Comment:
Same as the other test. This should probably be converted into a set of unit tests

Reply:
I separated the test un 4 tests as requested, check it out